### DISCUSSION TITLE
How to alias environment variable in Github Actions

### DISCUSSION THREAD

**Discussion by fbiville** [community/community#26358] [Actions]
<p>Hi,</p>
<p>I am trying to inject <code>GITHUB_WORKSPACE</code> value into another step environment variable to no avail.<br>
I have tried:</p>
<pre><code class="lang-yaml">    env:
        MY_ENV: ${{ env.GITHUB_WORKSPACE }}
</code></pre>
<p>and</p>
<pre><code class="lang-yaml">    env:
        MY_ENV: $GITHUB_WORKSPACE
</code></pre>
<p>… to no avail.</p>
<p>Is there a way to do this?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I just tried, the <code>env</code> context doesn’t seem to work there. Specifically for <code>GITHUB_WORKSPACE</code> you have an alternative by using the <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context"><code>github</code> context</a>:</p>
<pre><code class="lang-yaml">    env:
        MY_ENV: ${{ github.workspace }}
</code></pre>
